numpy/core/tests/test_internal.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import division, absolute_import, print_function
+
+import sys
+import numpy as np
+from numpy.testing import (
+    TestCase, assert_, run_module_suite, assert_array_equal)
+
+
+class CtypesAvailabilityTest(TestCase):
+
+    def setUp(self):
+        self.test_arr = np.array([[1, 2, 3], [4, 5, 6]])
+
+    def test_ctypes_is_available(self):
+        import ctypes
+        from numpy.core import _internal
+
+        _ctypes = _internal._ctypes(self.test_arr)
+        shape = _ctypes.get_shape()
+
+        self.assertEqual(ctypes, _ctypes._ctypes)
+        self.assertEqual(len(shape), 2)
```

Comment:
assert_equal(shape, [2, 3]) would be shorter here. Also, we seem to prefer `assert_equal` over `self.assertEqual`